annotations.proto

```diff
@@ -30,6 +30,8 @@ message MessageOptions {
   optional bool set = 2;
```

Comment:
While updating the docker image in `lorawan-stack`, please update this file in `api/third_party/github.com/TheThingsIndustries/protoc-gen-go-flags/annotations.proto`.